packages/components/src/components/types/icons.types.ts

```diff
@@ -0,0 +1,30 @@
+export interface IIconsManifest {
+    // If a new icon category is added, be sure to add it here as well. This is part of the fix for ts(7053)
+    readonly brand: string;
+    readonly cashier: string;
+    readonly common: string;
+    readonly contract: string;
+    readonly currency: string;
+    readonly dxtrade: string;
+    readonly flag: string;
+    readonly mt5: string;
+    readonly option: string;
+    readonly stock: string;
+    readonly tradetype: string;
+    readonly underlying: string;
+    readonly wallet: string;
+}
+export type TIconProps = {
+    className?: string;
```

Comment:
the previous type also included `object` . don't we need it any more?

Reply:
@iman-fs, Only string were being passed in here and by convetion, classNames are strings. the object applies when you are using the `classnames` library, in which case, it's result would be a string as well. e.g `classNames({'test1': is_test1, 'test2': is_test2})` would return `'test1 test2'` as the new value for the className prop. So we I believedon't need it